src/stories/RefAnchoredOverlay.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {useState, useRef} from 'react'
+import {Button, Text, ButtonDanger, Position, Flex, BaseStyles, ThemeProvider} from '..'
+import {Meta} from '@storybook/react'
+
+import {AnchoredOverlay} from '../AnchoredOverlay'
+
+export default {
+  title: 'Internal components/AnchoredOverlay',
+  component: AnchoredOverlay,
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ]
+} as Meta
+
+export const RefAnchoredOverlay = () => {
+  const [isOpen, setIsOpen] = useState(false)
+  const buttonRef = useRef<HTMLButtonElement>(null)
+  const confirmButtonRef = useRef<HTMLButtonElement>(null)
+  const anchorRef = useRef<HTMLDivElement>(null)
+  const closeOverlay = () => setIsOpen(false)
+  return (
+    <Position position="absolute" top={0} left={0} bottom={0} right={0} ref={anchorRef}>
+      <Button ref={buttonRef} onClick={() => setIsOpen(!isOpen)}>
```

Comment:
Would it be better to have the button as the anchor?  Or some other separate element that really shows off the disconnection between the open button and the menu positioning.